### ISSUE TITLE
NoSuchMethodError error loading Dreamscape server in Parabots client

### ISSUE THREAD

**Issue by UFOSrReal** [Parabot/Parabot#161]
* Provide a general summary of the issue in the **Title** above.  
* Before you open an issue, please check if a similar issue already exists or has been closed before.

### Important
Mark with [x] to select. Leave as [ ] to unselect.

### When reporting a bug/issue:
- [ ] Screenshot
- [ ] The Parabot version and the operating system you're using
- [ ] The behavior you expect to see, and the actual behavior
- [ ] Steps to reproduce the behavior 
- [ ] \(optional) Possible solution/fix/workaround

### When you open an issue for a change/improvement/feature request:
- [ ] A description of the problem you're trying to solve, including _why_ you think this is a problem
- [ ] If the feature changes current behavior, reasons why your solution is better
- [ ] \(optional) Possible solution/fix/workaround

### Description
**Mac, client 2.6.3**
Getting error in both IDE and run.bat when loading DreamScapes server in parabots client using:

`java -jar -noverify Parabot.jar`

Error in terminal:

```
java.lang.NoSuchMethodError: org.parabot.core.asm.redirect.RuntimeRedirect.exec(Ljava/lang/Runtime;[Ljava/lang/String;)Ljava/lang/Process;
	at com.dreamscape.SerialAddress.generateSerialNumber(SerialAddress.java:46)
	at com.dreamscape.SerialAddress.retainSerialNumber(SerialAddress.java:30)
	at com.dreamscape.GameClient.<clinit>(GameClient.java:47)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance0(Native Method)
	at sun.reflect.NativeConstructorAccessorImpl.newInstance(NativeConstructorAccessorImpl.java:62)
	at sun.reflect.DelegatingConstructorAccessorImpl.newInstance(DelegatingConstructorAccessorImpl.java:45)
	at java.lang.reflect.Constructor.newInstance(Constructor.java:423)
	at java.lang.Class.newInstance(Class.java:442)
	at org.rev317.min.Loader.fetchApplet(Loader.java:42)
	at org.parabot.core.Context.load(Context.java:181)
	at org.parabot.environment.servers.executers.ServerExecuter$1.run(ServerExecuter.java:28)
	at java.lang.Thread.run(Thread.java:745)
```

**Comment by CapslockRSPS** [comment]
Seemslike they are trying to get the users serial address and it

**Comment by JKetelaar** [comment]
`exec` is being blocked by Parabot, we'll have to create some custom code to let the client think it actually succeeded this step ;)

**Comment by JKetelaar** [comment]
https://github.com/Parabot/Parabot/blob/master/src/main/java/org/parabot/core/asm/redirect/RuntimeRedirect.java#L18

**Comment by JKetelaar** [comment]
Fixed in pull request #166
